packages/main/src/Tab.js

```diff
@@ -113,6 +113,14 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
+		 * Used to switch the active state (pressed or not) of the component.
+		 * @public
```

Comment:
Should be private. It will still behave as public - it will still cause invalidation, etc.
But we don't want to introduce "active" as a public API.